.buildkite/release-builds.yml

```diff
@@ -17,20 +17,23 @@ steps:
 
   - label: ":wordpress: :testflight: WordPress Release Build (App Store Connect)"
     command: ".buildkite/commands/release-build-wordpress.sh $BETA_RELEASE"
+    priority: 1
```

Comment:
Nit: I wonder if it wouldn't make sense to have the TestFlight build have `priority: 2` even, to be prioritized over the AppCenter build below? 🤷

Reply:
Makes sense! I updated the PR to put 2 as the priority for the TestFlight builds.